Lib/test/test_typing.py

```diff
@@ -5283,10 +5283,12 @@ class Node(Generic[T]): ...
                   Tuple[Any, Any], Node[T], Node[int], Node[Any], typing.Iterable[T],
                   typing.Iterable[Any], typing.Iterable[int], typing.Dict[int, str],
                   typing.Dict[T, Any], ClassVar[int], ClassVar[List[T]], Tuple['T', 'T'],
-                  Union['T', int], List['T'], typing.Mapping['T', int]]
+                  Union['T', int], List['T'], typing.Mapping['T', int],
+                  Union[b"x", b"y"]]
         for t in things + [Any]:
```

Comment:
I know this predates your PR, but can't we just include `Any` directly in the list here?

```suggestion
                  Union[b"x", b"y"], Any]
        for t in things:
```